content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -32,19 +32,34 @@ mitigate some future vulnerabilities too.
 {{% thirdparty-content single="true" %}}
 <!-- if adding another runtime in the future, omit the single setting -->
 
-This is a Linux only feature. In addition, support is needed in the 
+This is a Linux-only feature and support is needed in Linux for idmap mounts on
+the filesystems used. This means:
+
+* The filesystem you use for `/var/lib/kubelet/pods/`, or the custom directory
+  you configure for this, needs idmap mount support.
+* All the filesystems used in the pod need support for idmap mounts.
```

Comment:
```suggestion
* Every volume that is mounted into any container in the pod must support idmap mounts.
```

Reply:
This smells slightly incorrect. What supports idmap mount is the filesystem, not the volume. It doesn't matter if it is an EBS volume, a local disk volume or what. The only thing that matters is that the filesystem used in that volume supports idmap mounts in the kernel running on the host.

So, what about this instead?
```
* All the filesystems used in the pod's volumes must support for idmap mounts.
```

Reply:
Changed it to that, let me know what you think